deploy/ccm/secret-manager-role.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   namespace: kube-system
-  name: secret-manager
+  name: px-telemetry-secret-manager
```

Comment:
Did we not remove this role and role-binding, right?

Reply:
keep it here for now but it's not used. can be removed later.